### PR TITLE
Update anchors-in-regular-expressions.md

### PR DESCRIPTION
**Pull request by noctwolf**
## Summary

More accurate description

### CODE REVIEW

**Comment by noctwolf** [comment]
dotnet/docs.zh-cn#437

File: docs/standard/base-types/anchors-in-regular-expressions.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ ms.custom: seodec18
 |Pattern|Description|  
 |-------------|-----------------|  
 |`^`|Begin the match at the beginning of the input string (or the beginning of the line if the method is called with the <xref:System.Text.RegularExpressions.RegexOptions.Multiline?displayProperty=nameWithType> option).|  
-|`((\w+(\s?)){2,}`|Match one or more word characters followed either by zero or by one space exactly two times. This is the first capturing group. This expression also defines a second and third capturing group: The second consists of the captured word, and the third consists of the captured spaces.|  
+|`((\w+(\s?)){2,}`|Match one or more word characters followed either by zero or by one space at least two times. This is the first capturing group. This expression also defines a second and third capturing group: The second consists of the captured word, and the third consists of the captured white-space.|
```
Comment (rpetrusha) [inline docs/standard/base-types/anchors-in-regular-expressions.md]:
```suggestion
|`((\w+(\s?)){2,}`|Match one or more word characters followed either by zero or by one space at least two times. This is the first capturing group. This expression also defines a second and third capturing group: The second consists of the captured word, and the third consists of the captured white space.|  
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for making these corrections, @noctwolf. I've suggested one change: "white space" as a noun consists of two words. See #4439.

**Comment by noctwolf** [comment]
@rpetrusha Thank you for your guidance

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**